### PR TITLE
WebP quality/lossless compression parameters

### PR DESCRIPTION
**Pull request by mozgsml**
- WebP quality/lossless compression parameters
- Add WebP to the documentation
- Fix 2 wand tests

### CODE REVIEW

File: tests/test_wand.py [hunk 1]
```diff
@@ -201,11 +201,25 @@ def test_open_webp(self):
...
     def test_open_webp_w_alpha(self):
         with open('tests/images/tux_w_alpha.webp', 'rb') as f:
-            image = PillowImage.open(WebPImageFile(f))
+            image = WandImage.open(WebPImageFile(f))
 
         self.assertTrue(image.has_alpha())
         self.assertFalse(image.has_animation())
 
+    @unittest.skipIf(no_webp_support,
+                     "Pillow does not have WebP support")
```
Comment (kaedroho) [inline tests/test_wand.py:210]: Pillow => Imagemagick

File: tests/test_wand.py [hunk 2]
```diff
@@ -201,11 +201,25 @@ def test_open_webp(self):
...
+    def test_open_webp_quality(self):
+        high_quality = self.image.save_as_webp(io.BytesIO(), quality=90)
+        low_quality = self.image.save_as_webp(io.BytesIO(), quality=30)
+        self.assertTrue(low_quality.f.tell() < high_quality.f.tell())
+
+    @unittest.skipIf(no_webp_support,
+                     "Pillow does not have WebP support")
+    def test_open_webp_lossless(self):
```
Comment (kaedroho) [inline tests/test_wand.py:218]: Could we check that the pixel data stays the same?

Comment (mozgsml) [reply tests/test_wand.py:218]: Pixel-wise comparison for Wand takes about 10 seconds. The RGB part of completely transparent pixels is different, so I skipped them.

**Comment by kaedroho** [comment]
Looks good to me, thanks again!

**Comment by kmohrf** [comment]
Would you consider releasing this as a new version? I’m considering to add WEBP support to a wagtail project, but the lack of control over quality makes it unfeasible as the default quality is too bad in my case.

**Comment by kaedroho** [comment]
@kmohrf  Just released 1.4!

**Pull request merged**